apps/civil/civil-rtl-export/civil-rtl-export.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: civil-rtl-export
+  namespace: civil
+spec:
+  releaseName: civil-rtl-export
+  values:
+    java:
+      image: hmctspublic.azurecr.io/civil/rtl-export:prod-a6b5f16-20241007123558 #{"$imagepolicy": "flux-system:civil-rtl-export"}
```

Comment:
Add `useInterpodAntiAffinity: true` before the image property.  Doesn't seem to be mentioned in the documentation, but seemingly all of the other project specify it.